lib/util/transport.js

```diff
@@ -148,6 +151,42 @@ var qlb_split = function (buffer, print) {
     return ret;
 };
 
+var decode_packed_xml_contents = function(xmlBuf) {
+    var el, partName;
+    var res = {};
```

Comment:
A list of objects? I used object to save not just content, but also it's name (extracted from HTML element name and it's attributes).